doc/_utils/redirects.py

```diff
@@ -104,19 +101,13 @@
     ("config_and_build/bootloaders_and_dfu/index", "app_dev/bootloaders_dfu/index"),
     ("config_and_build/bootloaders_dfu/index", "app_dev/bootloaders_dfu/index"),
     ("config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib"), # MCUboot and NSIB (landing)
-    ("ug_bootloader_testing", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib"), # Testing the bootloader chain (removed after 2.7.0)
     ("app_dev/bootloaders_and_dfu/bootloader_testing", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib"),
     ("config_and_build/bootloaders_and_dfu/bootloader_testing", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib"),
     ("config_and_build/bootloaders/bootloader_testing", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_mcuboot_nsib"),
     ("config_and_build/bootloaders/bootloader_quick_start", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_quick_start"), # Quick start guide
     ("config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_quick_start", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_quick_start"),
     ("config_and_build/bootloaders/bootloader_adding_sysbuild", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild"), # Enabling a bootloader chain using sysbuild
     ("config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild"),
-    ("ug_bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"), # "Enabling a bootloader chain using child and parent images (deprecated)"
-    ("app_dev/bootloaders_and_dfu/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
-    ("config_and_build/bootloaders_and_dfu/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
-    ("config_and_build/bootloaders/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
-    ("config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
```

Comment:
```
    ("ug_bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"), # "Enabling a bootloader chain using child and parent images (removed in v3.0.0)"
    ("app_dev/bootloaders_and_dfu/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
    ("config_and_build/bootloaders_and_dfu/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
    ("config_and_build/bootloaders/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
    ("config_and_build/bootloaders_dfu/mcuboot_nsib/bootloader_adding", "app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding"),
```
@annwoj , what should these redirects point to?

Reply:
Since they're both about enabling a bootloader chain, I think it makes most sense to redirect it to sysbuild page (bootloader_adding_sysbuild)